entries.py

```diff
@@ -308,30 +246,11 @@ def generate_plaintext(self):
 	pronunciation="", 
 	seealso="")
 
-YAML = Gloss("YAML", 
+YAML = GlossEntry("YAML", 
 	acronym_full="YAML Ain't Markup Language", 
 	definition="Human-readable data-serialization lanaguage. Commonly used for configuration files.", 
 	furtherreading="https://yaml.org/", 
 	institute="", 
 	internal=False, 
 	pronunciation="", 
-	seealso="")
-
-# there's gotta be a way to do this automagically
-
-for entry in [
-	AnVIL,
-	BDC, BDCat, BioDataCatalyst,
-	CancerGenomicsCloud, CGC, CommonWorkflowLanguage, Cromwell, CWL,
-	DAG, Docker, DockerContainer, Dockerfile, DockerImage, DockerLayer, DockstoreCLI, DOI,
-	Elwazi,
-	JSON,
-	NCI, NHGRI, NHLBI, NIH
-	OICR,
-	Terra,
-	WDL, Workflow_Description_Language, 
-	UCSC, 
-	YAML]:
-	entry.generate_plaintext()
```

Comment:
I understand not wanting to do it this way... the `gc.get_objects()` feels kinda sketchy to me. An option I prefer would be:

```
import entries

for entry in entries dir(entries):
    if isinstance(glossary_object, GlossEntry):
        etc...
```

but my favorite way to do this would be to just define all of the entries in a big list in entries.py.

```
glossary = [
    GlossaryEntry('foo'),
    GlossaryEntry('bar')
]
```

This approach is nice because you don't need to redundantly name the variables.

Reply:
@jessebrennan I tried a few iterations of your first example but it's having issues with glossary_object, ie

> import entries
> for entry in dir(entries):
>     if isinstance(glossary_object, GlossEntry):
>         glossary_object.generate_plaintext()

doesn't work. I had previously tried a few iterations of similar syntax but I couldn't find a way to pass in user-defined objects without using gc.